### PR TITLE
Update editorconfig

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: src/Utilities/Compiler/Options/Unit.cs [hunk 1]
```diff
@@ -45,7 +45,9 @@ internal struct Unit : IEquatable<Unit>
         /// <param name="first">The first <see cref="Unit"/> value to compare.</param>
         /// <param name="second">The second <see cref="Unit"/> value to compare.</param>
         /// <returns>Because <see cref="Unit"/> has a single value, this always returns <c>true</c>.</returns>
-        [SuppressMessage("Microsoft.Usage", "CA1801:ReviewUnusedParameters", MessageId = "first", Justification = "Parameter required for operator overloading."), SuppressMessage("Microsoft.Usage", "CA1801:ReviewUnusedParameters", MessageId = "second", Justification = "Parameter required for operator overloading.")]
+        [SuppressMessage("Microsoft.Usage", "CA1801:ReviewUnusedParameters", MessageId = "first", Justification = "Parameter required for operator overloading.")]
+        [SuppressMessage("Microsoft.Usage", "CA1801:ReviewUnusedParameters", MessageId = "second", Justification = "Parameter required for operator overloading.")]
+        [SuppressMessage("Style", "IDE0060:Remove unused parameter", Justification = "Parameter required for operator overloading.")]
```
Comment (Evangelink) [inline src/Utilities/Compiler/Options/Unit.cs]: It'd be nice to have `CA1801` and `IDE0060` mutually exclusive.

Comment (mavasani) [reply src/Utilities/Compiler/Options/Unit.cs]: Yep, need a suppressor.

**Comment by mavasani** [comment]
@sharwell Should this PR target 2.9.x branch and then flow to master through auto-merge?

**Comment by sharwell** [comment]
@mavasani I don't think it would make much difference, but I can retarget it if you want. Let me know what you prefer.

**Comment by mavasani** [comment]
@dotpaul?

**Comment by dotpaul** [comment]
If it's easy enough to retarget to 2.9.x, I'd say do it, just to make merging easier.

**Comment by sharwell** [comment]
@mavasani @dotpaul this is now retargeted to 2.9.x

**Pull request merged**